modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+If the current viewport is too small to present the entire menu, access to the ‘overflow’ is ensured by presenting a scrollbar in the menu.
+
+Previously, however, the rendering of this menu’s height (which, in turn, establishes whether a scroll bar is required) was done with the wrong element. Rendering was done by `tox-autocompleter`, not `tox-menu`. (`tox-menu` renders the height of all other {productname} menus.)
+
+In some circumstances this error resulted in the top or bottom menu borders to scroll out of view, making the meny appear unusable.
+
+This update corrects this. Rendering of the `autocompleter` menu height is now assigned to `tox-menu` and its menu borders do not scroll out of view.
+
+
+=== Link menu and toolbar buttons now always execute the `mceLink` command
+
+Selecting *Insert > Link…* or clicking the Link toolbar button did not, previously, execute the `mceLink` command. Equivalently, if the `link_quicklink` option was enabled, using the Ctrl+K or ⌘+K shortcut to open the Contextual Link Toolbar did not execute `mceLink` either.
+
+This was not consistent with other, related, actions. For example, clicking the Unlink toolbar button executed the `unlink` command. As well, not executing a command meant it was not possible to track the *Link…* dialog being opened, and the command could not be over-ridden with `addCommand`.
+
+With this update, `mceLink` is always executed when the Link dialog or Contextual Link Toolbar is opened.
+
+
+=== Choice menu items will now use the `menuitemradio` ARIA role to better reflect that only a single item can be active
+
+{productname} uses the https://w3.org/WAI/ARIA/[Accessible Rich Internet Applications] (ARIA) Working Group guidelines to make {productname} accessible to as wide a range of users as is possible.
+
+To this end, some {productname} menu items were assigned the `menuitemcheckbox` ARIA role. However, screen readers report this role as meaning each item in the menu can be toggled independently. This is not true: the options in these menus are mutually exclusive.
+
+The appropriate ARIA role for these objects is `menuitemradio` and, as of this release, this is the ARIA role these menu items are assigned.
+
+Consequently, screen readers will, correctly, report the options in these menus as mutally exclusive.
+
+
+[[improvements]]
+== Improvements
+
+{productname} 6.1 also includes the following improvements:
+
+=== All help dialog links to `https://tiny.cloud` now include `rel="noopener"` to avoid potential security issues
+
+`rel` is an attribute for the `<a>` element (and other elements). And setting this attribute to `rel="noopener"`  instructs the browser to load the resource referenced in the anchor tag in the same browsing context (ie, it instructs the browser to not open a new window).
```

Comment:
This is incorrect? `rel="noopener"`, particularly when used with `target="_blank"`, instructs the browser to set the `window.opener` JavaScript API to `null`. By default it is a reference back to the page that opened the link. This is the security issue.

Before this change, developers were unknowingly trusting our website to not use this API to interfere with their website. We never would, but now we can't.

Reply:
Quoting my `git` commit messages is now a running theme:

Corrected errors in help-dialog-links-now-contain-rel=noopener-attribute-and-value write-up.